labgrid/config.py

```diff
@@ -196,3 +196,6 @@ def get_images(self):
             images[name] = self.resolve_path(image)
 
         return images
+
+    def get_features(self):
```

Comment:
The commit message doesn't match the method name.

Reply:
Fixed.